proto/pulumi/plugin.proto

```diff
@@ -42,4 +42,25 @@ message PluginDependency {
 // to, this message is used so the provider can still be passed the engine address to communicate with.
 message PluginAttach {
     string address = 1; // the grpc address for the engine
+}
+
+message PackageParameterization {
```

Comment:
Slightly annoyingly this is exactly the same as "Parameterization" in resources.proto but we can't move it, and we can't circularly depend on it. Might be they need to diverge in the future anyway.